### PR TITLE
Add proxy support

### PR DESCRIPTION
**Pull request by zzl221000**
feat: #10

### CODE REVIEW

File: src-tauri/src/models/settings.rs [hunk 1]
```diff
@@ -97,8 +102,15 @@ pub fn read(db: &Connection, key: &SettingKey) -> Result<Setting> {
...
-    Ok(rows.next()?.map(Setting::from).unwrap())
+    match rows.next()? {
+        None => {
+            Err(Error::Unknown)
+        }
+        row => {
+            Ok(row.map(Setting::from).unwrap())
+        }
+    }
+    // Ok(rows.next()?.map(Setting::from).unwrap())
```
Comment (parksb) [inline src-tauri/src/models/settings.rs:113]: It looks like this line can be deleted.

**Review by parksb — COMMENTED** [review]
Thank you for your work! I've left a few comments :)

File: src-tauri/src/commands/feeds.rs [hunk 2]
```diff
@@ -8,8 +8,8 @@ use crate::{
 };
 
 #[tauri::command]
-pub fn create_feed(db_state: State<DbState>, arg: FeedToCreate) -> Result<String, String> {
-    let title = match fetch_feed_title(&arg.link) {
+pub fn create_feed(db_state: State<DbState>, arg: FeedToCreate, proxy: Option<&str>) -> Result<String, String> {
```
[the remark this answers is not in the document]
Comment (zzl221000) [reply src-tauri/src/commands/feeds.rs]: You're right.
Taking the value from the database is better, I have changed it.

**Comment by parksb** [comment]
Thank you :)

**Pull request merged**